react/components/ProductName/index.js

```diff
@@ -100,32 +100,16 @@ class ProductName extends Component {
       )
     }
 
-    const productNameClasses = classNames(`${productName.productNameContainer} mv0`, {
-      [`${className}`]: className
-    })
-
-    const productBrandClasses = classNames(`${productName.productBrand}`, {
-      [`${brandNameClass}`]: brandNameClass
-    })
-
-    const productSkuClasses = classNames(`${productName.productBrand}`, {
-      [`${skuNameClass}`]: skuNameClass
-    })
-
-    const productReferenceClasses = classNames(`${productName.productReference}`, {
-      [`${productReferenceClass}`]: productReferenceClass
-    })
-
     return (
-      <Wrapper className={productNameClasses}>
-        <span className={productBrandClasses}>
+      <Wrapper className={`${productName.productNameContainer}`}>
```

Comment:
You don't need string interpolation here. Just use `className={productName.productNameContainer}`, same as above cases